dfsclient/src/test/resources/generate-datasets.ecl

```diff
@@ -33,7 +36,9 @@ ds := DATASET(totalrecs1, transform(rec,
                                    self.r8 := (REAL)(random() % unique_values);
                                    self.r4 := (REAL)(random() % unique_values);
                                    self.dec16 := (REAL)(random() % unique_values);
+                                   self.dec15 := (REAL)(random() % unique_values);
                                    self.udec16 := (REAL)(random() % unique_values);
+                                   self.udec15 := (REAL)(random() % unique_values);
```

Comment:
Is there a corresponding test to verify these odd len decimals are correctly read?

Reply:
@rpastrana Yes, tests in DFSReadWriteTest read those generated datasets, write them back, and then compare the datasets against the generated datasets for correctness.